Dockerfile

```diff
@@ -6,9 +6,14 @@ ARG XX_VERSION=1.1.1
 ARG GOVERSIONINFO_VERSION=v1.3.0
 ARG GOTESTSUM_VERSION=v1.7.0
 ARG BUILDX_VERSION=0.8.2
+ARG OSXCROSS_VERSION=11.3-r7
 
 FROM --platform=$BUILDPLATFORM tonistiigi/xx:${XX_VERSION} AS xx
 
+FROM crazymax/osxcross:${OSXCROSS_VERSION}-alpine AS osxcross-alpine
```

Comment:
So, is this similar to https://github.com/docker/golang-cross (from your other comments?)